content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -0,0 +1,136 @@
+will suffer due to these sub-optimal allocations.
+
+The Topology Manager is a new Kubelet component, that acts as a source of truth for other Kubelet components to make topology aligned resource decisions.
+
+The Topology Manager provides an interface for components, called *Hint Providers*, to send and recieve topology information. The default *best-effort* algorithm takes
+all possible NUMA Node combinations for an incoming container and chooses the best fit (being the narrowest NUMA node with resources available). The topology manager receives 
+the NUMA information from the *Hint Providers* as a bitmask. The Topology Manager preforms a bitwise and to get the combined results. Finally, the Topology Manager selects the narrowest preffered result, and stores it.
+The *Hint Providers* then use this when making the resource allocation decisions.
+
+### Configuration
+
+The Topology Manager currently:
+* Works on a nodes with the `static` CPU Manager Policy enabled. See [control CPU Management Policies](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/)
+* Works on Pods in the `Guaranteed` QOS Class {{< glossary_tooltip text="QoS class" term_id="qos-class" >}}. 
+If these conditions are met, Topology Manager will align CPU and device requests.
+
+Topology Manager supports two allocation policies. You can set a policy via a Kubelet flag, `--topology-manager-policy`.
+There are two supported policies:
+
+*`none` (default): no Topology Management; Kubelet does not consult Topology Manager for placement decisions 
+*`best-effort`: which means the Topology Manager will provide the best fit allocation for the container
+but will not fail the pod in the event this is not possible.
+
+### None policy {#policy-none}
+
+This is the default policy and does not perform any topology alignment.
+
+### Best-effort policy {#policy-best-effort}
+
+For each container in a Guaranteed Pod, kubelet, with `best-effort` topology 
+management policy, calls each Hint Provider to discover their resource availability.
+Using this information, the Topology Manager stores the 
+best NUMA Node affinity for that container. If the affinity is not preferred, 
+Topology Manager will store this and admit the pod to the node.
+
+The *Hint Providers* can then use this information when making the 
+resource allocation decision.
+
+### Example Usage
+
+Consider the containers in the following pod specs:
+
+```yaml
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+```
+
+This pod runs in the `BestEffort` QoS class because no resource `requests` or
+`limits` are specified.
+
+```yaml
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    resources:
+      limits:
+        memory: "200Mi"
+      requests:
+        memory: "100Mi"
+```
+
+This pod runs in the `Burstable` QoS class because requests are less than limits.
+
+If you selected a `best-effort` policy, Topology Manager would not consider either of these Pod

```

Comment:
Nit (to keep it 3rd person):
With "best-effort" policy enabled, Topology Manager will not consider either of these Pod specifications.